src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const propArr = sourceString.split(';');
+  const result = {};
+
+  for (let i = 0; i < propArr.length; i++) {
+    const el = propArr[i].split(': ');
```

Comment:
```suggestion
    const [firstValue, secondValue] = propArr[i].split(': ');
```

You can destructure array here